man/ctags.1.rst.in

```diff
@@ -1842,6 +1842,10 @@ TMPDIR
 FILES
 -----
 
+$XDG_CONFIG_HOME/ctags/\*.ctags (on other than MSWindows)
+
+$HOME/.config/ctags/\*.ctags (on other than MSWindows)
```

Comment:
Why wouldn't `~/.config/ctags` be supported on Windows? It would simplify your documentation and the logic to allow that. And it is convenient for cross-platform users who share configs across machines.